sql/mysql/convert.go

```diff
@@ -84,7 +84,7 @@ func FormatType(t schema.Type) (string, error) {
 	case *schema.SpatialType:
 		f = strings.ToLower(t.T)
 	case *schema.TimeType:
-		f = strings.ToLower(t.T)
+		f = fmt.Sprintf("%s(%d)", strings.ToLower(t.T), t.Precision)
```

Comment:
What if the precision was not defined by the user (i.e. 0)? Do you prefer to format it as `timestamp(0)` instead of `timestamp`?